react/Footer/FooterLink/FooterLink.js

```diff
@@ -36,9 +38,16 @@ FooterLink.propTypes = {
     PropTypes.arrayOf(PropTypes.node),
     PropTypes.node
   ]),
-  linkRenderer: PropTypes.func.isRequired
+  linkRenderer: PropTypes.func.isRequired,
+  authedHref: PropTypes.string,
```

Comment:
I agree that the name doesnt fit the function at the moment. It sounds like it takes a different href rather than a boolean. What are your thoughts on something like `authRequired`?